compiler/ast.nim

```diff
@@ -553,7 +553,7 @@ type
     tfIterator,       # type is really an iterator, not a tyProc
     tfPartial,        # type is declared as 'partial'
     tfNotNil,         # type cannot be 'nil'
-    tfRequiresInit,   # type constains a "not nil" constraint somewhere or
+    tfRequiresInit,   # type constrains a "not nil" constraint somewhere or
```

Comment:
```suggestion
    tfRequiresInit,   # type contains a "not nil" constraint somewhere or
```

Reply:
I think... it's "constrains contraint" instead of "contains contraint"

Reply:
It's contains, constraining a constraint doesn't make sense and "containing somewhere" is more legible than "constraining somewhere'